sdk/include/token.h

```diff
@@ -38,6 +38,8 @@ __BEGIN_DECLS
  */
 SKey __cheri_compartment("alloc") token_key_new(void);
 
+typedef __cheri_callback void *(*Token_Allocator)(Timeout *, SObj, size_t);
```

Comment:
This is odd capitalisation.  Doesn't clang-tidy tell you to remove the underscore?